docs/elasticsearch.md

```diff
@@ -0,0 +1,204 @@
+# Elasticsearch
+
+With Inago, it's a piece of cake to deploy an Elasticsearch cluster,
+as well as updating it without any downtime or data loss.
+
+It should be noted that this is a demo, and there are dangers to using it in a production environment without further testing.
+Saying that, if you want to do so, and have questions, the team is always available on IRC (`#giantswarm` on `freenode`) <3
+
+## Setup
+- We're using a CoreOS cluster running locally, with Vagrant. See https://coreos.com/os/docs/latest/booting-on-vagrant.html for further information.
```

Comment:
Hmm, we should firstly link to our coreos provisioning stuff and maybe even to onsho before we mention coreos vagrant. (not that I don't want to mention the coreos vagrant option, just want our options to be there, too). But open to other thoughts.